### PR TITLE
[WIP] Metaphysics connection

### PR DESCRIPTION
**Pull request by erikdstock**
This PR should ideally set up connections both through the built site pages and components requiring authentication that can't be built immediately.

Some current data coming in via the REST api:

### CODE REVIEW

File: src/utils/gravity.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const response = await fetch(
+      `${apiUrl}/me/bidder_positions?sort=-created_at`,
+      {
+        headers: { "x-access-token": jwt },
+      }
+    )
+    const json = await response.json()
+    return json
+  }
+}
```
Comment (damassi) [inline src/utils/gravity.ts]: @erikdstock - why not with something like [gatsby-source-appiserver](https://www.gatsbyjs.org/packages/gatsby-source-apiserver)?

File: src/components/account/BidsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const last4 = (s: string) => {
+  const length = s.length
+  return s.slice(length - 5, length)
+}
+
+export const BidsPage: RouteComponent = () => {
+  const [bids, setBids] = useState<any[] | null>(null)
+  if (!bids) {
+    gravity.meBidderPositions().then(data => setBids(data))
```
Comment (damassi) [inline src/components/account/BidsPage.tsx]: Wouldn't this be better pulled through metaphysics?

File: src/utils/gravity.ts (hunk 1 above)
Comment (damassi) [reply src/utils/gravity.ts]: Ah yeah duh, static files ahead of time...

File: src/components/account/BidsPage.tsx (hunk 2 above)
Comment (erikdstock) [reply src/components/account/BidsPage.tsx]: gatsby can't build or run currently when connecting to metaphysics via `gatsby-source-graphql` - though we could still be making queries like this the old fashioned way. good point.

**Comment by erikdstock** [comment]
merging this to clear the way for actual metaphysics work

**Pull request merged**